elfo-test/src/proxy.rs

```diff
@@ -88,15 +88,25 @@ impl Proxy {
     #[track_caller]
     pub fn recv(&mut self) -> impl Future<Output = Envelope> + '_ {
         static STD_CLOCK: Lazy<StdClock> = Lazy::new(StdClock::new);
+        static TIMER_SERVICE: Lazy<Arc<TimerService>> = Lazy::new(|| {
+            let timer_service = Arc::new(TimerService::new(&*STD_CLOCK));
+            thread::spawn({
```

Comment:
Maybe it's better just to spawn a task?

Reply:
And how make it non busy loop task as we can't use tokio::sleep here?

Reply:
You're right.